content/en/hub/intro/release-notes/tn-12_0-beta2.md

```diff
@@ -10,6 +10,8 @@ linkTitle: "TrueNAS 12.0-BETA2 and Hotpatches"
 TrueNAS 12.0-BETA2.1 is a hotpatch release to fix a ZFS permissions issue that affects the base FreeBSD OS ([NAS-107270](https://jira.ixsystems.com/browse/NAS-107270)).
 12.0-BETA2 users are encouraged to update to BETA2.1 as soon as possible.
 
+[CVE-2020-24717](https://security.ixsystems.com/cves/2020-08-27-cve-2020-24717/)
```

Comment:
```suggestion
[CVE-2020-24716](https://security.ixsystems.com/cves/2020-08-27-cve-2020-24716/) and [CVE-2020-24717](https://security.ixsystems.com/cves/2020-08-27-cve-2020-24717/)
```

Reply:
Even though there was one fix, there were two unique details so MITRE assigned two CVEs.

Reply:
I see it now, thanks for catching that.